src/sidebar/components/Annotation/AnnotationPublishControl.js

```diff
@@ -81,14 +82,17 @@ function AnnotationPublishControl({
   return (
     <div className="AnnotationPublishControl">
       <div className="annotation-publish-button">
-        <Button
-          className="annotation-publish-button__primary"
+        <LabeledButton
+          className="PublishControlButton"
```

Comment:
I wonder if we might want to extract a component out of this pattern of button + dropdown menu in future.

Reply:
Yep; that's part of the reason I have the custom styling in the catch-all `styles/sidebar/buttons.scss` module—so they can be reviewed later. See also #3259 for a pattern-to-maybe-extract-in-future.